### ISSUE TITLE
{"status":"fail","reason":"no_routes","retry_after":3600}

### ISSUE THREAD

**Issue by quadermamun1** [perezdidac/WhatsAPINet#311]
I'm getting the bellow error:
{"status":"fail","reason":"no_routes","retry_after":3600}

By executing the bellow URL either by .NET code or by web browser.
https://v.whatsapp.net/v2/code?cc=968&in=xxxxxxxx&to=xxxxxxxxxx&method=sms&sim_mcc=422&sim_mnc=002&token=QgbvsIkCMn7jZJXT2oLHXmX3ajM%3D&id=%bf%aa%90%2d%00%80j%c4d3%81%e9%b8%a1%ed%d9%8fpn%ea&lg=ar&lc=OM

Please help

**Comment by lkathke** [comment]
I've got the same error, after I got "un-authorized" as errormessage and wanted to reregister my number. -.-

**Comment by nasurinhakimu** [comment]
Same goes to me. Please help, please.

**Comment by StandardLake** [comment]
@nasurinhakimu  do you have any solution to the non-authorised issue ?

**Comment by mgp25** [comment]
@StandardLake If you are blocked, you cant unblock your number.

**Comment by nasurinhakimu** [comment]
@StandardLake No, but I tried Wart 1.7.3 and it seems worked.

**Comment by StandardLake** [comment]
Can you please provide the link to the 1.7.3 version you are referring to

**Comment by nasurinhakimu** [comment]
This one: https://github.com/shirioko/WART
I used the latest version, which is 1.7.3.0
Does it fix your problem?

**Comment by StandardLake** [comment]
Yes, am able to register and get a password. Registering is not a problem
anymore.
Are you able to use the WhatsAppAPI ?
That one is not working at all. It keeps reporting "AUTH ERROR"

Any clue??

**Comment by nasurinhakimu** [comment]
Glad to hear it works.
About WhatsAppAPI, it does not work for me too.

**Comment by abdalla12002** [comment]
https://github.com/shirioko/WART
does not exist anymore

**Comment by nasurinhakimu** [comment]
@abdalla12002 do you require the WART application?
I have uploaded the necessary files here: https://sites.google.com/site/davivirtus/everyone

**Comment by mgp25** [comment]
https://github.com/mgp25/WART
